Orion.Net.Client/Scripts/BaseClientScript.cs

```diff
@@ -47,9 +67,23 @@ protected async Task SendStringContent(string contentResult)
 
             // Send result content to server :
             await connector.SendResultCommand(result);
+        }
 
-            // Notifiy server that result has been sent
+        /// <summary>
+        /// Post ImageContentResult, with file from path save as byte array, on the paltform
+        /// </summary>
+        /// <param name="pathImage"></param>
+        /// <returns></returns>
+        protected async Task SendImageContent(string pathImage)
+        {
```

Comment:
Please add some checks about the pathImage
null, StringOrWhitespace or perhaps that the file does not exists